sdk/examples/poet_python/sawtooth_poet/main.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+import sys
+import argparse
+import pkg_resources
+
+from sawtooth_sdk.processor.log import init_console_logging
+from sawtooth_sdk.processor.log import log_configuration
+from sawtooth_sdk.processor.config import get_log_dir
+
+from sawtooth_sdk.consensus.zmq_driver import ZmqDriver
+from sawtooth_poet.engine import PoetEngine
+
+DISTRIBUTION_NAME = 'sawtooth-poet'
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawTextHelpFormatter)
+
+    parser.add_argument(
+        '-C', '--connect',
+        default='tcp://localhost:5005',
```

Comment:
Do engines talk on the component port or is this 5005 going to be the new standard for a consensus port?

Reply:
This is provisional endpoint that will be added to the validator. We may merge it in with the component endpoint to simplify deployment, but I don't think we should commit to that until we understand how it will affect performance: https://github.com/hyperledger/sawtooth-core/pull/1692/commits/9f723237ab0ac315a4784202f73eda98582f2d58

Reply:
should this port be 5050 then?

Reply:
yep